R/drive_update.R

```diff
@@ -41,24 +47,85 @@ drive_update <- function(file,
     stop_collapse(c("Path to update is not unique:", files))
   }
 
+  if (!is_mine(file)) {
+    stop_glue("\nCan't update this file because you don't own it:\n  * {file$name}")
+  }
+
+  params <- list(
+    fileId = file$id,
+    fields = "*",
+    ...)
+
+  if (identical(names(params), c("fileId", "fields"))) {
+    response <- drive_update_media(file = file, media = media)
```

Comment:
I think we need to either:

  1. Push `as_dribble(list(process_response(...)))` down into these 3 functions. *I think this is my preference?*
  1. Rename them to something like `do_update_*()`.

because right now their names suggest they are like our other `drive_*()` functions, i.e. that they return a dribble. Yet they do not.